freeipa/src/test/java/com/sequenceiq/freeipa/util/KrbKeyTabGeneratorTest.java

```diff
@@ -0,0 +1,56 @@
+package com.sequenceiq.freeipa.util;
+
+import com.cloudera.thunderhead.service.usermanagement.UserManagementProto.ActorKerberosKey;
+import com.sequenceiq.cloudbreak.common.service.Clock;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class KrbKeyTabGeneratorTest {
+
+    private static final long MOCK_TIME = 1576171731141L;
+
+    @InjectMocks
+    KrbKeytabGenerator underTest;
+
+    @Mock
+    private Clock clock;
+
+    @Test
+    void testGenerateKeytabBase64() {
```

Comment:
it looks like to me there is only 1 happy path tested. Are you sure there isn't anything which can go wrong here? Like a parameter is wrong

Reply:
I think there's some more validation (i.e. has to be a valid workload user) we can add, but it might be at a higher level.